src/Rector/v11/v2/MigrateFrameModuleToSvgTreeRector.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\Rector\v11\v2;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\StaticCall;
+use PhpParser\Node\Scalar\String_;
+use PHPStan\Type\ObjectType;
+use Rector\Core\Rector\AbstractRector;
+use Ssch\TYPO3Rector\Helper\FilesFinder;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @changelog https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/11.2/Deprecation-93944-FileTreeAsIframeMigratedToSVG-basedTree.html
+ * @see \Ssch\TYPO3Rector\Tests\Rector\v11\v2\MigrateFrameModuleToSvgTreeRector\MigrateFrameModuleToSvgTreeRectorTest
+ */
+final class MigrateFrameModuleToSvgTreeRector extends AbstractRector
+{
+    public function __construct(
+        private FilesFinder $filesFinder
+    ) {
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [StaticCall::class];
+    }
+
+    public function refactor(Node $node): ?Node
+    {
+        $fileInfo = $this->file->getSmartFileInfo();
+
+        if (! $this->filesFinder->isExtTables($fileInfo)) {
+            return null;
+        }
+
+        if (! $this->nodeTypeResolver->isMethodStaticCallOrClassMethodObjectType(
+            $node,
+            new ObjectType('TYPO3\CMS\Extbase\Utility\ExtensionUtility')
+        )
+            &&
+            ! $this->nodeTypeResolver->isMethodStaticCallOrClassMethodObjectType(
+                $node,
+                new ObjectType('TYPO3\CMS\Core\Utility\ExtensionManagementUtility')
+            )
+        ) {
+            return null;
+        }
+
+        if (! (property_exists($node, 'name') && null !== $node->name && (property_exists(
+            $node,
+            'args'
+        ) && null !== $node->args))) {
+            return null;
+        }
+
+        if (! $this->isNames($node->name, ['addModule', 'registerModule'])) {
+            return null;
+        }
+
+        $hasAstBeenChanged = false;
+        if ($this->isName($node->name, 'addModule')) {
+            if (! (property_exists($node->args[4], 'value') && null !== $node->args[4]->value)) {
```

Comment:
You could use the ValueResolver to get the value from the arguments so you could drop this additional checks here.